modules/timeutil/since.go

```diff
@@ -208,43 +114,14 @@ func timeSincePro(then, now time.Time, lang translation.Locale) string {
 	return strings.TrimPrefix(timeStr, ", ")
 }
 
-func timeSince(then, now time.Time, lang translation.Locale) string {
-	return timeSinceUnix(then.Unix(), now.Unix(), lang)
-}
-
-func timeSinceUnix(then, now int64, lang translation.Locale) string {
-	lbl := "tool.ago"
-	diff := now - then
-	if then > now {
-		lbl = "tool.from_now"
-		diff = then - now
-	}
-	if diff <= 0 {
-		return lang.Tr("tool.now")
-	}
-
-	_, diffStr := computeTimeDiff(diff, lang)
-	return lang.Tr(lbl, diffStr)
-}
-
-// TimeSince calculates the time interval and generate user-friendly string.
+// TimeSince renders relative time HTML given a time.Time
 func TimeSince(then time.Time, lang translation.Locale) template.HTML {
-	return htmlTimeSince(then, time.Now(), lang)
+	timestamp := then.UTC().Format(time.RFC3339)
+	// declare data-tooltip-content attribute to switch from "title" tooltip to "tippy" tooltip
+	return template.HTML(fmt.Sprintf(`<relative-time class="time-since" prefix="%s" datetime="%s" data-tooltip-content data-tooltip-interactive="true">%s</relative-time>`, lang.Tr("packages.versions.on"), timestamp, timestamp))
```

Comment:
Can we move translation key `packages.versions.on` to just `on`? If we leave this as-is, it works but will confuse anyone coming across this code. @wxiaoguang how would you suggest this to be done? Do it in all translation files manually?

Reply:
Well… The typical workflow is simply changing it in `locale_en-US.ini`, and the rest needs to be re-translated in Crowdin

Reply:
This `on` is not friendly to i18n. So I prefer to leave it as it is.

The plan for i18n is: https://github.com/go-gitea/gitea/issues/23863

Reply:
Okay, should keep this in mind for that then. I'll add a comment.

Reply:
Actually, I think I will add this root `on` now, it seems cleaner, even if it forces a (hopefully quick) re-translation.

Reply:
That's fine by me

Reply:
If we want to add a root `on`, it's better to make it have a long & clear name (clarify it's for "on date"). The `"on"` in code is too short and difficult to be searched / grepped.

Reply:
Yeah let's call it `on_date`.

Reply:
Done. I do think we need some helper utility in the future to rename a translation key while preserving existing translations.